tests/kernel/mem_protect/mem_protect/src/mem_domain.c

```diff
@@ -48,9 +48,9 @@ void test_mem_domain_setup(void)
 	struct k_mem_partition *parts[] = { &ro_part, &ztest_mem_partition };
 
 	num_rw_parts = max_parts - PARTS_USED;
-	zassert_true(NUM_RW_PARTS >= num_rw_parts,
-		     "CONFIG_MAX_DOMAIN_PARTITIONS incorrectly tuned, %d should be at least %d",
-		     CONFIG_MAX_DOMAIN_PARTITIONS, max_parts);
+	zassert_true(num_rw_parts <= NUM_RW_PARTS,
+			"CONFIG_MAX_DOMAIN_PARTITIONS incorrectly tuned, %d should be at least %d",
+			CONFIG_MAX_DOMAIN_PARTITIONS, max_parts);
```

Comment:
This is functionally the same. Why is this being changed?

Reply:
Hi @dcpleung,  @maksimmasalski  was trying to fix the warning message of checkpatch.pl:
```
tests/kernel/mem_protect/mem_protect/src/mem_domain.c:51: WARNING:CONSTANT_COMPARISON: Comparisons should place the constant on the right side of the test
#51: FILE: tests/kernel/mem_protect/mem_protect/src/mem_domain.c:51:
+       zassert_true(NUM_RW_PARTS >= num_rw_parts,

tests/kernel/mem_protect/mem_protect/src/mem_domain.c total: 0 errors, 1 warnings, 483 lines checked
```
So I think this change acceptable, right? thanks!